experimenter/experimenter/settings.py

```diff
@@ -124,6 +126,7 @@
             os.path.join(BASE_DIR, "legacy", "legacy-ui", "templates"),
             os.path.join(BASE_DIR, "nimbus-ui", "templates"),
             os.path.join(BASE_DIR, "docs"),
+            os.path.join(PROJECT_DIR, "templates"),
```

Comment:
The template directory should be in BASE_DIR/templates following [Django best practices](https://docs.djangoproject.com/en/4.2/howto/overriding-templates/)

Reply:
We have the other apps inside the experimenter directory contrary to what the default django project structure is where they are present at  the root, so the `BASE_DIR` refers to that path. I followed this structure which was suggested in an earlier feedback which will refer to the templates folder present at the root of the project like so. Should I move it back inside the experimenter directory ? // @brennie 
```project_name/
    manage.py
    project_name/
        __init__.py
        settings.py
        urls.py
        ...
    templates/
        base.html
        app1/
            template1.html
            template2.html
        app2/
            template3.html
            template4.html
    app1/
        __init__.py
        models.py
        views.py
        ...
    app2/
        __init__.py
        models.py
        views.py
        ...
    ...```

Reply:
`templates` should be located at `experimenter/experimenter/`. The first directory is just to separate the project from other projects. The second directory is the Python package directory.